src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -15,7 +15,7 @@ <h4>{{ data.name }} </h4>
             </div>
         </div>
         <div class="content-type__edit-actions">
-            <button type="button" pButton flat label="Edit" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
+            <button type="button" pButton secondary label="Edit" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
```

Comment:
We don't have "raised" and "flat" buttons, now we have: normal button (no need for class or attr), secondary attr and tiny attr, tiny and secondary can be combined.